File: src/main/java/de/maggiwuerze/xdccloader/events/channel/server/ChannelDeleteEvent.java
package de.maggiwuerze.xdccloader.events.channel.server;

import de.maggiwuerze.xdccloader.events.EntityDeleteEvent;
import de.maggiwuerze.xdccloader.model.entity.Channel;

public class ChannelDeleteEvent extends EntityDeleteEvent<Channel> {

	public ChannelDeleteEvent(Object source, Channel channel) {
		super(source, channel);
	}
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/channel/server/ChannelDeleteEvent.kt
package de.maggiwuerze.xdccwebloader.events.channel.server

import de.maggiwuerze.xdccwebloader.events.EntityDeleteEvent
import de.maggiwuerze.xdccwebloader.model.entity.Channel

class ChannelDeleteEvent(source: Any, channel: Channel) : EntityDeleteEvent<Channel>(source, channel)